static/src/js/providers/AuthContextProvider/AuthContextProvider.jsx

```diff
@@ -54,9 +53,21 @@ const AuthContextProvider = ({ children }) => {
     }
   }, [loginInfo])
 
+  const handleRefreshToken = (refreshToken) => {
+    if (process.env.NODE_ENV === 'production') {
+      const { tokenValue } = refreshToken
+      console.log('refreshing token ', tokenValue)
```

Comment:
Same comment about possibly testing this in the network tab, instead of having code that needs to be removed later